spinalcordtoolbox/reports/qc.py

```diff
@@ -456,13 +569,20 @@ def make_content_path(self):
         :return: return "root folder of the report" and the "furthest folder path" containing the images
         """
         # make a new or update Qc directory
-        target_img_folder = os.path.dirname(self.qc_params.abs_bkg_img_path())
-
-        try:
-            os.makedirs(target_img_folder, exist_ok=True)
-        except OSError as err:
-            if not os.path.isdir(target_img_folder):
-                raise err
+        if self.qc_params.command in ['sct_fmri_moco', 'sct_dmri_moco']:
+            target_img_folder = os.path.dirname(self.qc_params.abs_bkg_gif_path())
+            try:
+                os.makedirs(target_img_folder, exist_ok=True)
+            except OSError as err:
+                if not os.path.isdir(target_img_folder):
+                    raise err
+        else:
+            target_img_folder = os.path.dirname(self.qc_params.abs_bkg_img_path())
+            try:
+                os.makedirs(target_img_folder, exist_ok=True)
+            except OSError as err:
+                if not os.path.isdir(target_img_folder):
+                    raise err
```

Comment:
If `abs_bkg_gif_path` turns out not to be necessary, then I think this whole change is also unnecessary, since the existing code will be sufficient.